frontend/src/cards/ui/Sources.tsx

```diff
@@ -32,7 +32,20 @@ export function getDatasetIdsFromResponses(
   );
 }
 
-function getDataSourceMapFromDatasetIds(
+export const stripCountyFips = (datasetIds: string[]) => {
+  datasetIds.map((id) => {
+    if (/\d/.test(id)) {
```

Comment:
can we add a comment here just quickly explaining what it's doing and why? I hate trying to read REGEX lol